### PR TITLE
List Component

### PR DESCRIPTION
**Pull request by serling**

### CODE REVIEW

File: components/list/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+Generic list component (stateless). Generates a `<ul>` or `<ol>` with inline or block elements, through component parameters. Children of this component will be wrapped in `<li>` elements.
+
+Uses the BEM naming convention (http://getbem.com/) for class names.
+
+## Autor:
+
+Thomas Christoffersen
+
+## Dependencies:
```
[the remark this answers is not in the document]
Comment (asbjornh) [reply components/list/README.md]: There's no double bookkeeping as far as I can see? Regardless, this is the pattern used by all of the existing components in the library and if it should be addressed we should have a separate issue for it and update the entire library

File: components/list/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# List
+
+Generic list component (stateless). Generates a `<ul>` or `<ol>` with inline or block elements, through component parameters. Children of this component will be wrapped in `<li>` elements.
+
+Uses the BEM naming convention (http://getbem.com/) for class names.
```
Comment (asbjornh) [inline components/list/README.md]: Good that this is documented 👍

File: components/list/list.jsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        {child}
+      </li>
+    ))
+  );
+
+List.propTypes = {
+  inline: PropTypes.bool,
+  ordered: PropTypes.bool,
+  className: PropTypes.oneOfType([PropTypes.object, PropTypes.string]),
+  children: PropTypes.oneOfType([
```
Comment (asbjornh) [inline components/list/list.jsx]: I think that this is probably unnecessary and that `PropTypes.node` is enough. Can you check?

Comment (serling) [reply components/list/list.jsx]: You are right! Done.

**Pull request merged**